mscore/inspector/inspector.cpp

```diff
@@ -861,6 +864,23 @@ void InspectorRest::tupletClicked()
             }
       }
 
+//---------------------------------------------------------
```

Comment:
Why not put it in its own file like most other inspector panels?

Reply:
I could, but it would be pretty small, and many of the smaller ones are here.